MIGRATION-V1.md

```diff
@@ -0,0 +1,217 @@
+Here are simple steps to move from `v0.11.0` to `v1.0.0`.
+
+## AUTHENTICATION MECHANISM
+
+The constructor no longer accepts individual credentials like `IAMApiKey`, etc. We initialize authenticators from the [core](https://github.com/IBM/go-sdk-core). The core supports various authentication mechanisms, choose the one appropriate to your instance and use case.
+
+For example, to pass a IAM apikey:
+#### Before
+```go
+service, serviceErr := servicev1.
+NewServiceV1(&servicev1.ServiceV1Options{
+    URL: "{url}",
+    IAMApiKey: "{apikey}",
+})
+```
+
+#### After(V1.0)
+```go
+import "github.com/IBM/go-sdk-core/core"
+
+authenticator := &core.IamAuthenticator{
+    ApiKey:     "{apikey}",
+}
+
+service, serviceErr := servicev1.
+NewServiceV1(&servicev1.ServiceV1Options{
+    URL: "{url}",
+    Authenticator: authenticator,
+})
+```
+
+There are 5 authentication variants supplied in the SDK (shown below):
```

Comment:
I think we should consider re-using (i.e. linking to) the Authentication.md file that's in the go core repo in some fashion.  I think it would make sense for you to include sdk-specific authentication info in the go-sdk's documentation, but also link to the "reference" info that is in the go core.   That way, we have less duplication of documentation which (hopefully) will increase our chances of keeping it up to date and in sync.